### DISCUSSION TITLE
Translating the compendium items

### DISCUSSION THREAD

**Discussion by pawelkuklinski** [Muttley/foundryvtt-shadowdark#848] [General]
Hi
I looked at the `i18n` files and there are no entries for spells, gear, etc. Those are placed in the `/data/packs/` location and cannot be translated with a provided system. Is this intentional (probably caused by a license agreement with the Arcane Library) or do you plan to allow translating those descriptions too?

**Comment by Muttley** [comment]
Unfortunately you can't translate compendium items as easily as interface strings in Foundry, but there is a module called Babele that can be used to create your own language mappings for compendium items: https://foundryvtt.com/packages/babele

**Reply by pawelkuklinski** [reply]
Ok, thanks for the info
